python/lsst/ts/vanward/check_conda_package_versions.py

```diff
@@ -107,7 +107,7 @@ def main(opts):
         print("Done. All packages were found with the provided version.")
 
 
-if __name__ == "__main__":
+def runner():
```

Comment:
Line 79 needs to be changed to
`line = line + f"={xml_version}" + f"_{sal_version}"`